fabtests/functional/cq_data.c

```diff
@@ -43,6 +43,10 @@ static int run_test()
 	size_t size = 1000;
 	struct fi_cq_data_entry comp = {0};
 	struct fi_rma_iov remote;
+	uint64_t mask = -1;
```

Comment:
Please use UINT64_MAX instead of assigning a signed value to an unsigned variable.

Reply:
Done. Thank you!